### PR TITLE
Update broken links for edge extensions

### PR DESCRIPTION
**Pull request by timkimadobe**
Create reusable alert component for dynamic dependency warning

## Description

This PR updates:
1. Broken links for Edge extensions:
    1. Edge Network extension
    2. Consent for Edge Network
    3. Identity for Edge Network
2. Creates a reusable alert component for inline android gradle dynamic dependency warning (to benefit from the define once, reuse anywhere Adobe I/O component system)
    * Created new alerts.md in general `resources` directory 
    * **Would like to get feedback on the naming convention for these reusable alerts, please see implementation for current approach; should be specific to type of alert, platform, the alert text, etc.**

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/documentation/edge-network/tabs/index.md [hunk 1]
```diff
@@ -10,11 +12,7 @@ implementation 'com.adobe.marketing.mobile:edge:2.+'
 implementation 'com.adobe.marketing.mobile:edgeidentity:2.+'
 ```
 
-<InlineNestedAlert variant="warning" header="false" iconPosition="left">
-
-Using dynamic dependency versions is **not** recommended for production apps. Please read the [managing Gradle dependencies guide](../resources/manage-gradle-dependencies.md) for more information. 
-
-</InlineNestedAlert>
+<Alerts query="platform=android-gradle&componentClass=InlineNestedAlert"/>
```
Comment (emdobrin) [inline src/pages/documentation/edge-network/tabs/index.md:15]: uuuu, can we use this everywhere now? 😍
and can we have something similar for the registerExtension API and use for all extensions?

File: src/pages/documentation/identity-for-edge-network/tabs/faq.md [hunk 2]
```diff
@@ -8,13 +10,9 @@ implementation 'com.adobe.marketing.mobile:edge:2.+'
...
 
-<InlineNestedAlert variant="warning" header="false" iconPosition="left">
-
-Using dynamic dependency versions is **not** recommended for production apps. Please read the [managing Gradle dependencies guide](../resources/manage-gradle-dependencies.md) for more information. 
-
-</InlineNestedAlert>
+<Alerts query="platform=android-gradle&componentClass=InlineNestedAlert"/>
 
-2. Import the Mobile Core and Edge extensions in your application class.
+1. Import the Mobile Core and Edge extensions in your application class.
```
Comment (emdobrin) [inline src/pages/documentation/identity-for-edge-network/tabs/faq.md:15]: should this be 2?

File: src/pages/documentation/edge-network/tabs/index.md (hunk 1 above)
Comment (calebk1) [reply src/pages/documentation/edge-network/tabs/index.md:15]: This is a thing we can do????? That's actually so cool LOL

Comment (calebk1) [reply src/pages/documentation/edge-network/tabs/index.md:15]: Yeah that makes a lot of sense then. If this works on localhost, I'm satisfied with that, and 100% makes things SO much easier for maintaining.

Comment (calebk1) [reply src/pages/documentation/edge-network/tabs/index.md:15]: @timkimadobe For variant tagging, yes, let's take that on Slack to discuss what the best way to do that will be. We can then add that to the repo's readme.

File: src/pages/documentation/identity-for-edge-network/faq.md [hunk 3]
```diff
@@ -91,9 +91,9 @@ Each identity extension has its own API to retrieve their respective ECIDs as we
 
 A: Use both `getSdkIdentities` and `getIdentities`
 
-To get the identifiers used by the Adobe Solutions extensions, call [getSdkIdentities](../mobile-core/api-reference.md#retrieving-stored-identifiers).
+To get the identifiers used by the Adobe Solutions extensions, call [getSdkIdentities](../mobile-core/api-reference.md#getsdkidentities).
 
-To get the identifiers used by the AEP Edge extensions, call [getIdentities](./api-reference.md#getidentities).
+To get the identifiers used by the Edge extensions, call [getIdentities](./api-reference.md#getidentities).
 
 ## Q: How can I clear all the identifiers from the SDK when using both AEP Edge extensions and Adobe Solutions extensions?
```
Comment (calebk1) [inline src/pages/documentation/identity-for-edge-network/faq.md:98]: "... when using both Edge extensions and..."

**Review by calebk1 — APPROVED** [review]
Just one miss for AEP usage. Looks good otherwise!

**Review by emdobrin — APPROVED** [review]

**Pull request merged**